core/src/main/java/com/scalar/db/api/TransactionCrudOperable.java

```diff
@@ -77,7 +81,8 @@ public interface TransactionCrudOperable {
 
   /**
    * Mutates entries of the storage through a transaction with the specified list of {@link
-   * Mutation} commands.
+   * Mutation} commands. Note that the conditions set in Mutation will be ignored. Please program
+   * such conditions in a transaction if you want to implement conditional mutation.
```

Comment:
Sorry, I don't understand how to set mutation in a transaction? You are not referring to `Put.withCondition` method right ?

Reply:
I meant even if you set conditions to the specified mutations (with Put.withCondition() and Delete.withCondition() method), they will be ignored in transactions.

Actually, this comment is already in the Javadocs for `put()` and `delete()`, but we missed it for `mutate()`, so I added it.